src/NzbDrone.Core/RootFolders/RootFolderService.cs

```diff
@@ -153,7 +154,34 @@ private List<UnmappedFolder> GetUnmappedFolders(string path)
             foreach (string unmappedFolder in unmappedFolders)
             {
                 var di = new DirectoryInfo(unmappedFolder.Normalize());
-                results.Add(new UnmappedFolder { Name = di.Name, Path = di.FullName });
+                var fileNfoName = String.Format("{0}\\tvshow.nfo", di.FullName);
```

Comment:
Use `Path.Combine` to join paths, which works on different operating systems, currently this would break on anything other than Windows.